tests/postman/newman/collection.json

```diff
@@ -269,7 +268,22 @@
               "script": {
                 "type": "text/javascript",
                 "exec": [
-                  "tests[\"Status code is 400\"] = responseCode.code === 400;"
+                  "if (responseCode.code === 200) {",
```

Comment:
I would recommend to do a different test for the token re-issue:
1) Test that the refresh token **is** equal to the old one (at least I am assuming that's how it works)
2) test that the new access token is **not** equal to the old one (also an assumption that it makes a new one with an old refresh token)
3) Afterwards do the access_token assignment to the environment variable
4) change test descriptions to match what we are testing